### ISSUE TITLE
LMDB Errors

### ISSUE THREAD

**Issue by santagada** [phiresky/ripgrep-all#101]
Hi, what does this two errors mean?

```
adapter: zip
Error: lmdb error: MDB_MAP_FULL: Environment mapsize limit reached

adapter: zip
Error: could not write to cache: LmdbError(MapFull)
```

I get this when going through a very large folder structure of zipped log files

**Comment by emwal** [comment]
Deleting the cache-files helped me. If this error is still relevant to you, you can try this:

From the README:
„Also remember to disable caching with --rga-no-cache or clear the cache (~/Library/Caches/rga on macOS, ~/.cache/rga on other Unixes, or C:\Users\username\AppData\Local\rga on Windows) to debug the adapters.“

**Comment by phiresky** [comment]
Fixed in 0.10
